src/coreclr/jit/lsrabuild.cpp

```diff
@@ -3071,6 +3071,16 @@ int LinearScan::BuildOperandUses(GenTree* node, regMaskTP candidates)
         return 1;
     }
 #endif // FEATURE_HW_INTRINSICS
+    if (node->OperIs(GT_MUL))
```

Comment:
Only applicable for `TARGET_ARM64`?

Reply:
yes, ifdefed to arm64. However, we might need it on xarch too if we introduce "unsafe math" mode.